README.md

```diff
@@ -1,13 +1,38 @@
 # rskswap-ui
 Project for Swaping RBTC to rWBTC independently from the claim (and the other way round too) 
+
+## Architecture Diagram
+![Architecture Diagram](architecture-diagram.png)
+
+
+## Use Cases
+
+### Swap WBTC to rBTC 
+This use case allows a user that has crossed WBTC/BTCB from multichain.org onto RSK (rWE to swap those tokens for RBTC, using a relayer that provides with the necessary liquidity an paid with the btc-pegged asset. The flow is described as in the following sequence diagram
+
```

Comment:
To be completely honest, I'd just put a simplified version of the sequence diagram, or directly the sequence diagram. The current architecture-diagram.png doesn't add too much value to the understanding of how it works.